prime-router/src/main/kotlin/cli/tests/LoadTestSimulator.kt

```diff
@@ -15,15 +15,10 @@ import kotlin.concurrent.thread
 import kotlin.system.measureTimeMillis
 
 /**
- * Simulator is intended to simulate a wide variety of heavy load situations.
- *
- * todo Separate this into a useful Utilities class, so that many different simulation tests can use it.
+ * This is a Utility meant to provide a library of useful functions for running simulation load tests.
+ * Child classes of this implement actual tests.
  */
-class Simulator : CoolTest() {
-    override val name = "simulator"
-    override val description = "Simulate a pattern of submissions to RS. "
-    override val status = TestStatus.LOAD
-
+abstract class LoadTestSimulator : CoolTest() {
     /**
```

Comment:
Simulator is now an abstract class, and the parent of ALL load related tests.  I have retired a bunch of old tests that don't use the Simulator.   Examples of child classes that implement Simulator are in LoadTests.kt.